irods/test/data_obj_test.py

```diff
@@ -412,8 +412,9 @@ def test_obj_does_not_exist(self):
 
 
     def test_create_from_invalid_path__250(self):
-        possible_exceptions = { ex.CAT_UNKNOWN_COLLECTION:  (lambda serv_vsn : serv_vsn >= (4,2,9)),
-                                ex.SYS_INVALID_INPUT_PARAM: (lambda serv_vsn : serv_vsn <= (4,2,8))
+        possible_exceptions = { ex.SYS_INVALID_INPUT_PARAM: (lambda serv_vsn : serv_vsn <= (4,2,8)),
+                                ex.CAT_UNKNOWN_COLLECTION:  (lambda serv_vsn : (4,2,9) <= serv_vsn < (4,3,0)),
+                                ex.SYS_INVALID_FILE_PATH:   (lambda serv_vsn : (4,3,0) <= serv_vsn)
```

Comment:
Just a personal thing: for inequality operations I prefer to have the constant on the right instead. Took me a second to understand that this is for versions greater than or equal to 4.3.0. It's fine as-is, though.

Reply:
I see your point but, just for now we'll keep it, since it has a logic flow to it.